tests/admin/load-tests.php

```diff
@@ -81,7 +81,7 @@ public function test_perflab_add_modules_page() {
 		remove_all_filters( 'plugin_action_links_' . plugin_basename( PERFLAB_MAIN_FILE ) );
 
 		// Does not register the page if the perflab_active_modules filter is used.
-		add_filter( 'perflab_active_modules', function() {} );
+		add_filter( 'perflab_active_modules', static function() {} );
```

Comment:
Looks like we can simplify this a bit...
```suggestion
		add_filter( 'perflab_active_modules', '__return_null' );
```